src/main/java/ch/heigvd/pro/b04/polls/Poll.java

```diff
@@ -49,4 +53,18 @@ public void addQuestion(Question newQuestion) {
       pollQuestions.add(newQuestion);
     }
   }
+
+  /**
+   * Add a new {@link Session} to this {@link Poll} instance.
+   *
+   * @param newSession session to add
+   */
+  public void addSession(Session newSession) {
+    newSession.getIdSession().setIdxPoll(this);
+    if (sessionSet == null) {
+      sessionSet = Stream.of(newSession).collect(Collectors.toSet());
```

Comment:
Do we need to create a `Stream` and then `collect` it, rather than a simple `Set` with no elements in it ?